### DISCUSSION TITLE
Frontend initiatives meeting: December 9, 2020

### DISCUSSION THREAD

**Discussion by varl** [dhis2/notes#185] [Meetings] [answered]
_When_: December 9, 2020 @ 15:00 CET
_Where_ : Remote

# Agenda

- General updates:
  - GitHub Discussions in the Notes repo!
  - Auto-merging coming soon in GitHub (https://github.com/dhis2/notes/issues/71)
- Updates from active working groups
  - Ports to App Platform
  - Feature toggling
  - Cypress
  - PWA
  - Plugin builds
- migration to yarn2 (experimental)

**Comment by amcgee** [comment]
This is a test

**Reply by varl** [reply]
this is also a test

**Reply by amcgee** [reply]
Threaded repllies!

**Comment by amcgee** [comment]
Another thing

**Reply by varl** [reply]
no!

**Comment by varl** [comment, marked as answer]
Meeting minutes: https://github.com/dhis2/notes/blob/master/decisions/2020/12/09-fe-initiatives-meeting.md
